google-cloud-examples/src/main/java/com/google/cloud/examples/translate/snippets/DetectLanguageAndTranslate.java

```diff
@@ -42,16 +42,19 @@ public static void main(String... args) {
     // environment variable
     Translate translate = TranslateOptions.getDefaultInstance().getService();
 
-    // Detect the language of some text
-    Detection detection = translate.detect("Hola");
+    // Text of an "unknown" language to detect and then translate into English
+    final String mysteriousText = "Hola Mundo";
+
+    // Detect the language of the mysterious text
+    Detection detection = translate.detect(mysteriousText);
     String detectedLanguage = detection.getLanguage();
 
-    // Translate some text
+    // Translate the mysterious text from Spanish to English
```

Comment:
It's odd to say it is "mysterious text" and also that it is Spanish at the same time (which means it's not so mysterious :-P) (maybe just remove the "from Spanish" part here)